ext.php

```diff
@@ -17,18 +17,20 @@ class ext extends \phpbb\extension\base
 	 */
 	public function is_enableable()
 	{
-		return $this->s9e_textformatter_installed() && !$this->s9e_mediamebed_installed();
+		return $this->phpbb_version_is_valid() && !$this->s9e_mediamebed_installed();
```

Comment:
Specific phpBB version is the most important requirement now that specific fixes and updates have been made to it for this extension. The previous check is no longer needed since, of course the library will be present in phpBB 3.2.x (otherwise the entire bbcodes system wouldn't function).